packages/labs/react/src/create-component.ts

```diff
@@ -157,28 +158,6 @@ export const createComponent = <I extends HTMLElement, E extends Events>(
     __forwardedRef?: React.Ref<unknown>;
   };
 
-  // Set of properties/events which should be specially handled by the wrapper
-  // and not handled directly by React.
-  const elementClassProps = new Set(Object.keys(events ?? {}));
-  for (const p in elementClass.prototype) {
-    if (!(p in HTMLElement.prototype)) {
-      if (reservedReactProperties.has(p)) {
-        // Note, this effectively warns only for `ref` since the other
-        // reserved props are on HTMLElement.prototype. To address this
-        // would require crawling down the prototype, which doesn't feel worth
-        // it since implementing these properties on an element is extremely
-        // rare.
-        console.warn(
```

Comment:
I think we should keep this warning. We can move this into a dev build in the future.